js/execution.js

```diff
@@ -307,13 +307,25 @@ window.WaterbearProcess = (function () {
     ReusableStrand.prototype = Object.create(Strand.prototype);
 
     /**
+     * Psuedo-instruction. This is how you can tell I've been up too late.
+     */
+    Object.defineProperty(ReusableStrand.prototype, 'firstEverInstructionNoReallyYouGuys', {
```

Comment:
Fair enough, but get some sleep and then rename to something more semantic. What information does the string argument carry?